package.json

```diff
@@ -3,11 +3,12 @@
   "description": "A dead simple starting point for custom elements.",
   "version": "1.0.0",
   "license": "Apache-2.0",
-  "repository": "https://github.com/Netflix/x-element",
+  "repository": "github:Netflix/x-element",
```

Comment:
I went through some of the npm documentation again and tried to further “modernize” this `package.json` — worth further scrutiny for sure though.